src/cmd/policy.go

```diff
@@ -184,15 +184,17 @@ Examples:
 		)
 		rs, err := rego.Eval(context.Background())
 		cobra.CheckErr(err)
-		b, err := json.Marshal(rs[0].Expressions[0].Value) //TODO: need more advanced handling of multiple things in json and reading from stdin
+		b, err := json.Marshal(rs[0].Expressions[0].Value) // TODO: need more advanced handling of multiple things in json and reading from stdin
 		cobra.CheckErr(err)
 
 		if outputFilePath == "-" {
 			fmt.Println(string(b))
 		} else {
 			main := newFile(outputFilePath, false)
 			defer main.Close()
-			main.WriteString(string(b))
+			if _, err := main.WriteString(string(b)); err != nil {
+				log.Error().Err(err).Msg("")
```

Comment:
We need to ensure this will not mess up the final output that the Ruby app might be expecting.  This command is used by opslevel-runner service detection jobs that the monolith uses.

Also we should always add a message to the `.Msg` at least describing the expected errors this might throw so in the future if we see that error message we can do a code search to find this error line.

Reply:
Good catch! Looks like I was rushed this part a bit.